modules/rpc/handler.go

```diff
@@ -110,7 +114,7 @@ func (p panicOnewayInboundMiddleware) HandleOneway(ctx context.Context, req *tra
 func panicRecovery(ctx context.Context) {
 	if err := recover(); err != nil {
 		stats.RPCPanicCounter.Inc(1)
-		ulog.Logger(ctx).Error("Panic recovered serving request", "error", errors.Errorf("panic in handler: %+v", err))
+		ulog.Logger(ctx).Error("Panic recovered serving request.", zap.Any("error", err))
```

Comment:
Is this a thing? Periods after messages? If so, we should document somewhere and communicate with my team.